evolver/calibration/actions.py

```diff
@@ -2,23 +2,37 @@
 from copy import deepcopy
 from abc import ABC, abstractmethod
 
+import pydantic
+
 
 class CalibrationAction(ABC):
+    class UserInput(pydantic.BaseModel): ...
+
     @abstractmethod
     def execute(self, state: Dict[str, Any], payload: Dict[str, Any] = None) -> Dict[str, Any]:
         pass
 
 
 class DisplayInstructionAction(CalibrationAction):
+    class UserInput(pydantic.BaseModel):
+        confirmation: bool = pydantic.Field(..., description="Confirm that the instruction has been completed")
```

Comment:
for this boolean case we can probably default it to true, so the logic below (`if not confirmation`) would be activated instead of a de-serialization error

Reply:
ah got it, nice, will change.

Reply:
sorry, I meant default to `false`!!